render.py

```diff
@@ -0,0 +1,127 @@
+from xml.etree import ElementTree as ET
+
+ns = {'ns': 'http://www.transunion.com/namespace/pfs/v4'}
+
+def render_print_image(data):
+    root = ET.fromstring(data)
+        
+    result = "TRANSUNION CREDIT REPORT\n\n"
+    result += "<FOR>           <SUB NAME>          <MKT SUB>  <INFILE>    <DATE>     <TIME> \n"
```

Comment:
do not put linebreaks in list items. `all(len(l) == 77 for l in result)` should always be True